lib/src/controls/surfaces/expander.dart

```diff
@@ -100,11 +99,6 @@ class Expander extends StatefulWidget {
   /// open and `false` when closed.
   final ValueChanged<bool>? onStateChanged;
 
-  /// The height of the header.
-  ///
-  /// Defaults to 48.0
-  final double headerHeight;
```

Comment:
Instead of removing this property, maybe we could change it to a `BoxConstraints`. What do you think?

Reply:
sure, go ahead.

Reply:
Actually I believe we should leave this to be determined by the passed header widget itself. If the user wants to pass advanced constraints, they can specify it in the header parameter.

Reply:
The Container parent of header has been appended with a BoxConstraints property which is passed minHeight (default is 48.0 matching earlier headerHeight). minHeight is also now an optional parameter of Expander.